From 3780bee0daab26359d3755038cf5650ed118c174 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Fri, 3 Nov 2017 14:38:24 +0000
Subject: [PATCH] trivial: Add another application to the blacklist

---
 src/plugins/gs-plugin-hardcoded-blacklist.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/gs-plugin-hardcoded-blacklist.c b/src/plugins/gs-plugin-hardcoded-blacklist.c
index 8656a61e..0203f140 100644
--- a/src/plugins/gs-plugin-hardcoded-blacklist.c
+++ b/src/plugins/gs-plugin-hardcoded-blacklist.c
@@ -57,6 +57,7 @@ gs_plugin_refine_app (GsPlugin *plugin,
 		"tracker-preferences.desktop",
 		"Uninstall*.desktop",
 		"wine-*.desktop",
+		"gnome-session-properties.desktop",
 		NULL };
 
 	/* not set yet */
-- 
2.14.3

